Packs/Vectra/pack_metadata.json

```diff
@@ -16,5 +16,6 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
-    ]
+    ],
+    "hidden": true
```

Comment:
Only after the other changes are merged and uploaded we can add this change otherwise they will not show.